### PR TITLE
Add activate and update endpoints for integration requests

### PR DESCRIPTION
**Pull request by Neeraj-Ghodla**

### CODE REVIEW

File: src/api/integration-requests.md [hunk 1]
```diff
@@ -54,8 +54,7 @@ Integration Requests enable terminal integrations and acceptance of asset types
   body ({
     accountId: 'Jaim1Cu1Q55uooxSens6yk',
     merchantId: '5ee0c486308f590260d9a07f',
-    type: 'epay',
-    terminalId: '11000021'
```
Comment (Neeraj-Ghodla) [inline src/api/integration-requests.md:58]: Create Integration Requests endpoint doesn't need terminalId.

File: src/api/integration-requests.md [hunk 2]
```diff
@@ -67,12 +66,6 @@ Integration Requests enable terminal integrations and acceptance of asset types
 | merchantId | String | The Merchant ID for the Integration Request.           |
 | type       | String | The type of the Integration Request.                   |
 
-{% h4 Optional Fields %}
-
-|   Field    |  Type  |                            Description                            |
-| :--------- | :----- | :---------------------------------------------------------------- |
-| terminalId | String | The payment system terminal ID. Required for NZ Epay integration. |
-
```
Comment (Neeraj-Ghodla) [inline src/api/integration-requests.md:75]: Same here.

File: src/api/integration-requests.md [hunk 3]
```diff
@@ -134,6 +127,74 @@ Integration Requests enable terminal integrations and acceptance of asset types
...
+  body ({
+    terminalId: '11000021'
+  })
+{% endreqspec %}
+
+{% h4 Optional Fields %}
+
+|   Field    |  Type  |                                           Description                                           |
+| :--------- | :----- | :---------------------------------------------------------------------------------------------- |
+| terminalId | String | ePay terminalId for the Integration Request. Required if Integration Request if of type `epay`. |
```
Comment (IsaacMurtagh) [inline src/api/integration-requests.md:147]: typo: Request is of type `epay`

**Review by IsaacMurtagh — APPROVED** [review]

File: src/api/integration-requests.md (hunk 1 above)
Comment (ncjones) [reply src/api/integration-requests.md:58]: @Neeraj-Ghodla "type" is mandatory though

File: src/api/integration-requests.md (hunk 3 above)
Comment (ncjones) [inline src/api/integration-requests.md:147]:
```suggestion
| terminalId | String | ePay terminalId for the Integration Request. Required if type is `epay`. |
```

**Review by ncjones — APPROVED** [review]

**Pull request merged**